scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/JobsController.java

```diff
@@ -626,6 +619,33 @@ public void onSuccess(String jsonString) {
                                                         });
     }
 
+    public void checkJobPermissionMethod(Job job, Label label, KeyValueGrid variablesGrid,
+            KeyValueGrid genericInformationGrid) {
+
+        SchedulerServiceAsync scheduler = Scheduler.getSchedulerService();
+        scheduler.checkJobPermissionMethod(LoginModel.getInstance().getSessionId(),
+                                           job.getId().toString(),
+                                           "getJobState",
+                                           new AsyncCallback<String>() {
+                                               public void onFailure(Throwable caught) {
+                                                   String msg = JSONUtils.getJsonErrorMessage(caught);
+                                                   LogModel.getInstance().logImportantMessage(
+                                                                                              "Failed to access REST server endpoint : " +
+                                                                                              msg);
+                                               }
+
+                                               @Override
```

Comment:
@Override is needed there?

Reply:
good remark. I think yes because all asycn rest calls in the same class override it. I will add it to onFailure